hugegraph-core/src/main/java/com/baidu/hugegraph/config/CoreOptions.java

```diff
@@ -575,7 +575,7 @@ public static synchronized CoreOptions instance() {
 
     public static final ConfigOption<Long> SNOWFLAKE_DATACENTER_ID =
             new ConfigOption<>(
-                    "snowflake.datecenter_id",
+                    "snowflake.datacenter_id",
```

Comment:
Config break

Reply:
yep, this config influence little, we should update the `doc` later (mark)